src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -54,7 +54,6 @@ public static boolean containsNumbers(String phone, String numbers) {
         requireNonNull(phone);
         requireNonNull(numbers);
 
-        checkArgument(numbers.length() >= 2, "numbers to check must be at least 2 digits");
         return phone.contains(numbers);
```

Comment:
May want to change 

```java
return phone.contains(numbers);
```

to

```java
return phone.startsWith(numbers);
```

so that find by number only gets people whose `phone` starts with `numbers`, rather than people whose `phone` contains `numbers` anywhere inside it

Reply:
Fixed accordingly